samples/SDCard/README.rst

```diff
@@ -1,4 +1,27 @@
 SD Card
 =======
 
-SDCard/FAT file usage and write benchmark.
+I couldn't get SDCard to work on a Wemos D1 mini with an SD Card adaptor. Always gave the same error. SD Card Error: FF.
```

Comment:
Please, remove this text from here. This file should contain helpful information describing what the sample is doing.
As such this information is very good as part of the PR ticket but not quite useful in the README file.

Reply:
Removed